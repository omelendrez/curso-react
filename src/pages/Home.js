import React from "react";

import Header from "../components/Headers";

const Home = () => {
  return (
    <>
      <Header title="Home page" />
      <div className="container" />
    </>
  );
};

export default Home;
