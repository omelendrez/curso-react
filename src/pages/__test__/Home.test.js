import React from "react";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import Home from "../Home";
import "@testing-library/jest-dom/extend-expect";

describe("Testing Login page", () => {
  it("It should match snapshot", () => {
    const component = renderer
      .create(
        <MemoryRouter>
          <Home />
        </MemoryRouter>
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
