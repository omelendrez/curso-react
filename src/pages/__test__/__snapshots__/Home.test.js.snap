// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Login page It should match snapshot 1`] = `
Array [
  <div
    className="headers"
  >
    <div>
      <div
        className="logo"
      >
        <img
          alt="Logo"
          src="https://cdn.dribbble.com/users/31864/screenshots/3666062/attachments/819877/2.jpg?compress=1&resize=800x600"
        />
      </div>
      <div
        className="titulo"
      >
        Home page
      </div>
      <div>
        9/4/2021
      </div>
    </div>
    <nav>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/todos"
            onClick={[Function]}
          >
            Todos
          </a>
        </li>
        <li>
          <a
            href="/contactos"
            onClick={[Function]}
          >
            Contactos
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  <div
    className="container"
  />,
]
`;
